src/plugins/dam/components/image-asset-fields/style.scss

```diff
@@ -5,3 +5,15 @@
 .dam-media-container > div > img {
     max-width: 100%;
 }
+
+.spinner-wrapper {
```

Comment:
@jalopez13 custom css, especially this generic should be avoided if possible.  use the admin-ui components and/or bootstrap utility classes.

Reply:
Removed!